spec/helpers/federal_register_documents_helper_spec.rb

```diff
@@ -2,7 +2,28 @@
 
 describe FederalRegisterDocumentsHelper do
   describe '#federal_register_document_comment_period' do
-    let(:document) { mock_model(FederalRegisterDocument) }
+    let(:document) do
+      mock_model(
+        FederalRegisterDocument,
+        document_type: "Notice",
```

Comment:
Let's use single instead of double quotes for `document_type` and `publication_date`

Reply:
curses rubymine turned rubocop off. Fixed it now.